### PR TITLE
Examples added

### PR DESCRIPTION
**Pull request by mkuculyma**
Summary of changes:
- added example with file publisher 
- added example with Influxdb publisher

### CODE REVIEW

File: examples/tasks/load-influxdb.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+                            "config": {
+                                "host": "localhost",
+                                "port": 8086,
+                                "database": "test",
+                                "user": "admin",
+                                "password": "admin"
+                            }
+                        }
+                    ],
+                    "config": null
```
Comment (kindermoumoute) [inline examples/tasks/load-influxdb.json]: .

File: examples/tasks/load-influxdb.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+                }
+            ],
+            "publish": null
```
Comment (kindermoumoute) [inline examples/tasks/load-influxdb.json]: .

File: examples/tasks/load-influxdb.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+            },
+            "config": {
+                "/intel/procfs/load": {
+                    "proc_path": "/proc"
+                }
+            },
+            "process": [
+                {
+                    "plugin_name": "passthru",
+                    "process": null,
```
Comment (kindermoumoute) [inline examples/tasks/load-influxdb.json]: .

File: examples/tasks/task.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+            },
+            "config": {
+                "/intel/procfs/load": {
+                    "proc_path": "/proc"
+                }
+            },
+            "process": [
+                {
+                    "plugin_name": "passthru",
+                    "process": null,
```
Comment (kindermoumoute) [inline examples/tasks/task.json]: .

File: examples/tasks/task.json [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+                    "publish": [
+                        {
+                            "plugin_name": "file",
+                            "config": {
+                                "file": "/tmp/published_load"
+                            }
+                        }
+                    ],
+                    "config": null
```
Comment (kindermoumoute) [inline examples/tasks/task.json]: .

File: examples/tasks/task.json [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+                }
+            ],
+            "publish": null
```
Comment (kindermoumoute) [inline examples/tasks/task.json]: .

File: examples/tasks/load-influxdb.json (hunk 3 above)
Comment (mkuculyma) [inline examples/tasks/load-influxdb.json]: fixed

File: examples/tasks/load-influxdb.json (hunk 1 above)
Comment (mkuculyma) [inline examples/tasks/load-influxdb.json]: fixed

File: examples/tasks/load-influxdb.json (hunk 2 above)
Comment (mkuculyma) [inline examples/tasks/load-influxdb.json]: fixed

File: examples/tasks/task.json (hunk 4 above)
Comment (mkuculyma) [inline examples/tasks/task.json]: fixed

File: examples/tasks/task.json (hunk 5 above)
Comment (mkuculyma) [inline examples/tasks/task.json]: fixed

File: examples/tasks/task.json (hunk 6 above)
Comment (mkuculyma) [inline examples/tasks/task.json]: fixed

**Pull request merged**